.github/CODEOWNERS

```diff
@@ -11,31 +11,72 @@
 
 /BASELINE_NOTES.md
 /.integrated_tests.yaml                               @CusiniM @cssherman @rrsettgast @wrtobin @castelletto1 @jhuang2601 @paveltomin
-/inputFiles/                                          @CusiniM @cssherman @jhuang2601 @rrsettgast
```

Comment:
you should have a default for the subdirectories that don't have a listing

Reply:
this change was not intended.

Reply:
fixed.